redbot/cogs/downloader/downloader.py

```diff
@@ -1207,3 +1237,31 @@ def cog_name_from_instance(self, instance: object) -> str:
             msg = _("This command is not provided by a cog.")
 
         await ctx.send(box(msg))
+
+    @staticmethod
+    def format_failed_repos(failed: Iterable[str]) -> str:
```

Comment:
This typing is incorrect, the function needs the object to be iterable and sized - this type hint should probably be `Collection` (can be imported from `typing` module):
```suggestion
    def format_failed_repos(failed: Collection[str]) -> str:
```
Remember to also update parameter type in docstring.